Kconfig.zephyr

```diff
@@ -399,6 +399,41 @@ config BOOTLOADER_ESP_IDF
 	  inside the build folder.
 	  At flash time, the bootloader will be flashed with the zephyr image
 
+config BOOTLOADER_BOSSA
```

Comment:
I feel we do need some other place for all those bootloader options, the top level Kconfig.zephyr file does not seem to be the right place for this.

You are just adding into something that already exists, so we can fix this later I guess.